### PR TITLE
Switch to tf 2.2

### PR DESCRIPTION
**Pull request by GauthierGar**
change requirements.txt pour utiliser tensorflow2.2.0, ok avec python3.8.2
@natct10

### CODE REVIEW

**Comment by natct10** [comment]
Getting troubles with scikit learn:
```sh
ERROR: back 0.1.4 has requirement scikit-learn==0.21.1, but you'll have scikit-learn 0.22.1 which is incompatible.
```

**Comment by natct10** [comment]
> Getting troubles with scikit learn:
> 
> ```shell
> ERROR: back 0.1.4 has requirement scikit-learn==0.21.1, but you'll have scikit-learn 0.22.1 which is incompatible.
> ```

But it seems to work:
```sh
Successfully installed astunparse-1.6.3 gast-0.3.3 tensorboard-2.2.2 tensorboard-plugin-wit-1.7.0 tensorflow-2.2.0 tensorflow-estimator-2.2.0
```

File: requirements.txt [hunk 1]
```diff
@@ -6,5 +6,5 @@ scikit-learn==0.22.1
 pandas==0.25.3
 seaborn==0.10.0
 loguru==0.4.1
-tensorflow==2.1.0
-ruamel.yaml~=0.16.10
\ No newline at end of file
+tensorflow==2.2.0
+ruamel.yaml~=0.16.10
```
Comment (Thomas-Galtier) [inline requirements.txt:10]: Using conda I had this change on line 10
```suggestion
ruamel.yaml==0.16.10
```

**Comment by bowni** [comment]
Address also #171 !

File: requirements.txt [hunk 2]
```diff
@@ -6,5 +6,5 @@ scikit-learn==0.22.1
 pandas==0.25.3
```
Comment (natct10) [inline requirements.txt:6]:
```suggestion
pandas==1.0.5
```

Comment (natct10) [reply requirements.txt:6]: I cannot add another suggestion, but I'd update numpy too with `numpy==1.19.0`

File: requirements.txt (hunk 1 above)
Comment (RomainGoussault) [reply requirements.txt:10]: Did you try with an updated version of conda @Thomas-Galtier ?

**Comment by RomainGoussault** [comment]
> Worked for me with `numpy==1.19.0`, `pandas==1.0.5` & `tensorflow==2.2.0`

@GauthierGar can you update the PR with @natct10 proposed changes ?

File: requirements.txt (hunk 1 above)
Comment (Thomas-Galtier) [reply requirements.txt:10]: Yes but I already had an up-to-date version of conda, so I still got a error.

**Comment by RomainGoussault** [comment]
@GauthierGar any updates on this PR ?

**Comment by GauthierGar** [comment]
> @GauthierGar any updates on this PR ?

I have update the PR with the 2 suggestions from @natct10 and @Thomas-Galtier 
There is a conflict with the branch master, I don't know what happened @RomainGoussault

**Comment by RomainGoussault** [comment]
> > @GauthierGar any updates on this PR ?
> 
> I have update the PR with the 2 suggestions from @natct10 and @Thomas-Galtier
> There is a conflict with the branch master, I don't know what happened @RomainGoussault

I fixed the conflicts :)

**Comment by GauthierGar** [comment]
So it means I can merge the PR?
How did you resolve the conflict, and what is a conflict? :smile:

**Comment by RomainGoussault** [comment]
Merge conflicts happen when you merge branches that have competing commits (there are changes on the same lines of code on the two branches), and Git needs your help to decide which changes to incorporate in the final merge

https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/about-merge-conflicts

**Pull request merged**